src/Raven.Client/Documents/Commands/Batches/PutAttachmentCommandData.cs

```diff
@@ -25,12 +25,19 @@ public PutAttachmentCommandData(string documentId, string name, Stream stream, s
             PutAttachmentCommandHelper.ValidateStream(stream);
         }
 
+        public PutAttachmentCommandData(string documentId, string name, Stream stream, string contentType, string changeVector, bool fromEtl) : this(documentId, name, stream, contentType, changeVector)
```

Comment:
internal and the other way around. execute this ctor from the public one

Reply:
ok